### PR TITLE
Add mention of webhooks frontend and how to create a secret key

### PR DESCRIPTION
**Pull request by Mario54**
@jpjoyal @dannytranlx @alavers @spasiu 

To be merged after https://github.com/smooch/smooch/pull/1093

### CODE REVIEW

**Comment by jugarrit** [comment]
This is probably good :+1:

File: source/rest/index.md [hunk 1]
```diff
@@ -385,7 +389,7 @@ smooch.webhooks.delete('55c8d9758590aa1900b9b9f6').then(() => {
 > Response:
 
 ```
-200 OK
+204 OK
```
Comment (mspensieri) [inline source/rest/index.md]: :(

Comment (Mario54) [inline source/rest/index.md]: Yup will change back

**Comment by dannytranlx** [comment]
:+1:

**Comment by alavers** [comment]
:+1:

**Comment by dannytranlx** [comment]
:tada:

**Comment by Mario54** [comment]
Will wait until the feature is shipped before merging.

**Pull request merged**